src/libraries/System.Memory/src/System/Buffers/SequenceReader.cs

```diff
@@ -49,6 +49,15 @@ public SequenceReader(ReadOnlySequence<T> sequence)
         /// </summary>
         public readonly ReadOnlySequence<T> Sequence { get; }
 
+        /// <summary>
+        /// The unread portion of the <see cref="Sequence"/>.
+        /// </summary>
+        public readonly ReadOnlySequence<T> UnreadSequence
+        {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is this really needed?

Reply:
I did to be compliant with `UnreadSpan` in the same file

Reply:
cc: @JeremyKuhne

Reply:
I wouldn't add it until we demonstrate it is really needed. In this case in particular the code for `Slice()` is non-trivial, so I doubt we'd find a situation where this would measure if it caused an inline that wasn't happening anyway.

Reply:
ok thank's